libs/assertion/tests/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if (NOT HPX_WITH_TESTS)
```

Comment:
We do more nowadays... (see for instance here: https://github.com/STEllAR-GROUP/hpx/blob/master/libs/preprocessor/tests/CMakeLists.txt#L6)

Reply:
Sorry I missed this change when I rebased